trade-finance-manager-ui/server/controllers/case/amendments/amendmentBankReviewDate.get.api-test.js

```diff
@@ -23,6 +23,14 @@ const user = {
   email: 'test@localhost',
 };
 
+const gefFacility = {
+  facilitySnapshot: { dates: { coverEndDate: '20 Oct 2020' }, type: FACILITY_TYPE.CASH },
+};
+
+const bssEwcsFacility = {
+  facilitySnapshot: { dates: { coverEndDate: '20 Oct 2020' }, type: FACILITY_TYPE.BOND },
```

Comment:
same as above. can these repeated dates be made a const and maybe be added to test-mocks?

Reply:
These dates no longer feature here (they weren't essential for these tests)